hepstats/splot/exceptions.py

```diff
@@ -0,0 +1,9 @@
+"""
+Specific exceptions for the `splot` submodule
+"""
+
+class ModelNotFittedToData(Exception):
+    """Exception class for model not fitted to data provided to compute sweights"""
+
+    def __init__(self, *args, **kwargs):
+        Exception.__init__(self, *args, **kwargs)
```

Comment:
```suggestion
```
This can just be left away I think. Maybe inherit from a RunTimeError? But no strong opinion.